src/Modules/Backend/Domain/User/Command/ChangeUserHandler.php

```diff
@@ -15,14 +16,37 @@ public function __construct(
         private readonly UserRepository $userRepository,
         private readonly UserPasswordHasherInterface $passwordHasher,
         private readonly EventDispatcherInterface $eventDispatcher,
+        private readonly GoogleAuthenticatorInterface $googleAuthenticator,
     ) {
     }
 
     public function __invoke(ChangeUser $changeUser): void
     {
         $user = User::fromDataTransferObject($changeUser);
         $user->hashPassword($this->passwordHasher);
+
+        if (!$changeUser->enableTwoFactorAuthentication) {
+            $user->setGoogleAuthenticatorSecret(null);
+            $user->setBackupCodes();
```

Comment:
I would add the null here explicit for readability

Reply:
fixed a39a2696a26619f4b5488efb353d73a8eb4a9eb9